programmers/난이도별/level01.이상한_문자_만들기/Jaewon0702.py

```diff
@@ -0,0 +1,12 @@
+def solution(s):
+    ss = ""
+    s = s.split(" ")
+    for n in s:
+        for i, st in enumerate(n):
+            ss += st.lower() if i % 2 else st.upper()
+        ss += " "
+    return ss[:-1]

```

Comment:
```suggestion
    return " ".join([ "".join([ st.lower() if i % 2 else st.upper() for i, st in enumerate(n)  ]) for n in s.split(" ")])
```
이렇게 표현할 수 있습니다.

![image](https://user-images.githubusercontent.com/18749057/103596098-7555f180-4f40-11eb-8719-836d03f4940d.png)